### PR TITLE
Add mobile styles to create post view

### PR DESCRIPTION
**Pull request by etormasa**
# Description
add styles to new post form

## Screenshots

image
image

### CODE REVIEW

File: app/assets/stylesheets/components/_form.scss [hunk 1]
```diff
@@ -139,3 +139,18 @@
         margin: 0 40px 0 40px;
     }
 }
+.form{
```
Comment (Zama15) [inline app/assets/stylesheets/components/_form.scss]: space between { and the class

**Review by angel23v — COMMENTED** [review]
sort alphabetically

File: app/assets/stylesheets/components/_form.scss [hunk 2]
```diff
@@ -139,3 +139,18 @@
...
+    display: flex;
+    flex-direction: row;
+    height: 60%;
+    margin: auto;
+    width: 90%;
+    border-radius: 10px;
+    padding: 10px;
+}
+
+.col_form{
```
Comment (Zama15) [inline app/assets/stylesheets/components/_form.scss]: space between { and the class

File: app/assets/stylesheets/components/_form.scss
Comment (angel23v) [inline app/assets/stylesheets/components/_form.scss:1]: sort alphabetically

File: app/assets/stylesheets/components/_buttons.scss
Comment (angel23v) [inline app/assets/stylesheets/components/_buttons.scss:1]: sort alphabetically

**Review by Zama15 — CHANGES_REQUESTED** [review]
i add the class .form in my PR, lets see if we do not overwrite our changes

File: app/views/posts/_form.html.erb
Comment (angel23v) [inline app/views/posts/_form.html.erb:1]: Please indentation this file

File: app/assets/stylesheets/components/_buttons.scss
Comment (juperameza) [reply app/assets/stylesheets/components/_buttons.scss:1]: Aren't they already?

File: app/assets/stylesheets/components/_form.scss [hunk 3]
```diff
@@ -139,3 +139,18 @@
...
 }
+.form{
+    background-color: $tertiary-color-light;
+    display: flex;
+    flex-direction: row;
+    height: 60%;
+    margin: auto;
+    width: 90%;
+    border-radius: 10px;
+    padding: 10px;
```
Comment (juperameza) [inline app/assets/stylesheets/components/_form.scss]: ORder alphabetically 
```suggestion
    background-color: $tertiary-color-light;
    border-radius: 10px;
    display: flex;
    flex-direction: row;
    height: 60%;
    margin: auto;
    padding: 10px;
    width: 90%;
```

File: app/assets/stylesheets/components/_buttons.scss
Comment (angel23v) [reply app/assets/stylesheets/components/_buttons.scss:1]: good eye mentor

**Review by juperameza — CHANGES_REQUESTED** [review]
Shouldnt it be better to use a rich text box due for the post body?
Or is it intended to be something short and descriptive like a title per say

File: app/assets/stylesheets/components/_form.scss (hunk 3 above)
Comment (etormasa) [reply app/assets/stylesheets/components/_form.scss]: done :)

File: app/assets/stylesheets/components/_form.scss (hunk 1 above)
Comment (etormasa) [reply app/assets/stylesheets/components/_form.scss]: done :+1:

File: app/assets/stylesheets/components/_form.scss (hunk 2 above)
Comment (etormasa) [reply app/assets/stylesheets/components/_form.scss]: done, and changed the class name :+1:

File: app/assets/stylesheets/components/_form.scss
Comment (etormasa) [reply app/assets/stylesheets/components/_form.scss:1]: done

File: app/views/posts/_form.html.erb
Comment (etormasa) [reply app/views/posts/_form.html.erb:1]: done

**Review by angel23v — APPROVED** [review]

**Review by Zama15 — APPROVED** [review]
LGTM

**Review by juperameza — APPROVED** [review]

**Pull request merged**